packages/ui/components/validation-message.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+
+import { useEffect, useState, type FC } from "react";
+
+type ValidationMessageProps = {
+  error: string;
+  isSubmitting?: boolean;
+};
+
+export const ValidationMessage: FC<ValidationMessageProps> = ({ error, isSubmitting = false }) => {
```

Comment:
- Remove FC and type props in function args directly.
- Convert `ValidationMessageProps` to an interface (afaik, this should be fixed by eslint/prettier)